<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_profile</class>
 <widget class="QMainWindow" name="Change_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تغییر پروفایل</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>اطلاعات پروفایل</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>تلفن</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>نام</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ایمیل</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>نام کاربری</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>370</width>
         <height>170</height>
        </rect>
       </property>
       <property name="title">
        <string>تغییر رمز عبور</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>رمز فعلی</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>رمز جدید</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>تأیید رمز جدید</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="current_password_lineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="new_password_lineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="confirm_new_password_lineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="save_button">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>430</y>
         <width>220</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>ذخیره تغییرات</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="phone_lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>220</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="firstname_lineEdit">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>137</y>
         <width>223</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lastname_lineEdit">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>137</y>
         <width>223</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>250</y>
         <width>223</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_lineEdit">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>360</y>
         <width>223</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>